examples/triton-docker-combined/triton.tf

```diff
@@ -0,0 +1,48 @@
+# See http://www.joyent.com/blog/introducing-hashicorp-terraform-for-joyent-triton
+
+provider "triton" {
+  account = "${var.triton_account}"
+  key = "${var.triton_key_path}"
+  key_id = "${var.triton_key_id}"
+  url = "${var.triton_url}"
+}
+
+provider "docker" {
+  host = "${var.docker_host}"
+  cert_path = "${var.docker_cert_path}"
+}
+
+resource "docker_image" "nginx" {
+  name = "nginx:latest"
+  keep_updated = true
+}
+
+resource "docker_container" "nginx" {
+  count = 1
+  name = "nginx-terraform-${format("%02d", count.index+1)}"
+  image = "${docker_image.nginx.latest}"
+  must_run = true
+
+  env = ["env=test", "role=test"]
+
+  ports {
+    internal = 80
+    external = 80
+  }
+}
```

Comment:
Do you want to add the Docker features that we'd removed to try and make this work with 0.6.8?

```json
  restart = "always"
  memory = 128

labels {
  env = "test"
  role = "docker"
}

log_driver = "json-file"
log_opts = {
  max-size = "1m"
  max-file = 2
}
```